spec/services/types_generator_spec.rb

```diff
@@ -160,4 +162,80 @@
       end
     end
   end
+
+  describe 'cocina mapping' do
```

Comment:
Add a block of tests to ensure cohesion between the work types and subtypes as defined in the `WorkType` model and the types and subtypes as declared in the genre and resource type mappings.

Reply:
cool, so this is the test we talked about to ensure we have a mapping to cocina for each user option?

Reply:
@peetucket yessir